homeassistant/components/plugwise/config_flow.py

```diff
@@ -44,12 +62,39 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     VERSION = 1
     CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
 
+    def __init__(self):
+        """Initialize the Plugwise config flow."""
+        self.discovery_info = {}
+
+    async def async_step_zeroconf(self, discovery_info: DiscoveryInfoType):
+        """Prepare configuration for a discovered Plugwise Smile."""
+        self.discovery_info = discovery_info
+        _properties = self.discovery_info.get("properties")
+
+        unique_id = self.discovery_info.get("hostname").split(".")[0]
+        await self.async_set_unique_id(unique_id)
+        self._abort_if_unique_id_configured()
+
+        _product = _properties.get("product", None)
+        _version = _properties.get("version", "n/a")
+        _name = f"{ZEROCONF_MAP.get(_product,_product)} v{_version}"
```

Comment:
There's a missing space after the comma. Didn't CI black catch this?

Reply:
Apparently neither did pre-commit then ... let me check

Reply:
I guess because it's inside the format that it doesn't trigger ... running black manually with/without space doesn't yield different results

Reply:
I'll quickly rescan as well, but opened #37457 in the meantime